packages/backend/src/core/discovery/proxies/auto/StarkWareProxy.ts

```diff
@@ -32,9 +31,13 @@ async function getCallImplementation(
   provider: DiscoveryProvider,
   address: EthereumAddress,
 ) {
-  return bytes32ToAddress(
+  const callImplementation = bytes32ToAddress(
     await provider.getStorage(address, CALL_IMPLEMENTATION_SLOT),
   )
+
+  if (callImplementation === EthereumAddress.ZERO) return
```

Comment:
I like the solution, but having ZERO address here is not a desired behaviour and probably the config is wrong. Maybe we should notify about this somehow, do you have any ideas?

Reply:
Yeah that is something I wanted to verify with you if zero address is even correct there

Reply:
Also, probably not the place to do it in code.